File: Python/Easy/3370_Smallest_Number_With_All_Set_Bits/solution.py
class Solution:
    def smallestNumber(self, n: int) -> int:
        binLength = 0
        while n != 0:
            binLength += 1
            n = n // 2
        
        count = 0
        for i in range(binLength):
            count += (2 ** i)
        
        return count

File: Golang/Easy/3370_Smallest_Number_With_All_Set_Bits/solution.go
package smallestnumberwithallsetbits

import "math"

func smallestNumber(n int) int {
	binLength := 0
	for n != 0 {
		binLength += 1
		n = n / 2
	}

	count := 0
	for i := range binLength {
		count += int(math.Pow(2, float64(i)))
	}
	return count
}
